src/spinner/spinner.ts

```diff
@@ -3,7 +3,7 @@ import { property } from 'lit/decorators.js';
 import { classMap } from 'lit/directives/class-map.js';
 
 export class Spinner extends LitElement {
```

Comment:
We should add the `size` property to the spinner element. since the small button uses ` --tap-sys-spacing-7` for the spinner size.